src/main/java/newbank/server/exceptions/CustomerMaxAccountsException.java

```diff
@@ -1,5 +1,5 @@
 package newbank.server.exceptions;
 
 public class CustomerMaxAccountsException extends Exception {
-
+  private static final long serialVersionUID = 1234564L;
```

Comment:
I don't think there is a point in adding them (regardless of the warnings) unless we are:
- utilizing them in serialization
- they are actually unique

Reply:
The IDE adds them in automatically if you allow it too. It also get rids of compiler warnings.

Kindest Regards,
Jules